apps/prairielearn/elements/pl-excalidraw/pl-excalidraw.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-env browser,jquery */
```

Comment:
I don't see jQuery used anywhere on this page.

```suggestion
/* eslint-env browser */
```